### DISCUSSION TITLE
standard_init_linux.go:219: exec user process caused: exec format error

### DISCUSSION THREAD

**Discussion by wvendrig** [itzg/docker-minecraft-server#802] [Q&A]
This is my first post on github and I hope I didn't miss something of rules or anything of that kind
I'm trying to use this docker minecraft server to load it on a ODROID with HASSIO installed. followed the steps from the read me but probably there is some incompatiblilty here
since after starting the container mc stopped and in the log I see
_standard_init_linux.go:219: exec user process caused: exec format error_

what can I do to get the minecraft server running here?

**Comment by itzg** [comment]
What image + tag are you using specifically? Are you running RaspberryPi or some other ARM platform?

**Comment by wvendrig** [comment]
Thanks for replying 
image + tag, Sorry really new to this not quite sure what you mean by this
I'm using an ODROID-N2+ https://www.hardkernel.com/shop/odroid-n2-with-4gbyte-ram-2/ If I believe this site it is an ARM platform:
The main CPU is based on big.Little architecture which integrates a quad-core ARM Cortex-A73 CPU cluster and a dual core Cortex-A53 cluster with a new generation Mali-G52 GPU.

I've installed Home Assistant Operating System which is a specialized OS for Home assistent a house automation tool to control lot's of things around the house. Since the ODROID has lot's of power I would think it's perfect to also run minecraft server for my children. 

This HASSIO has ways to install plugins from all kinds of repo's and there is a version of docker and portainer available. I thought I just could use this to roll out the minecraft server installation which I tried using your Github. However it seems to be a little bit more to it to get this running.

Any help would be appreciated!

**Reply by itzg** [reply]
Yes, that helps a lot. Try using `itzg/minecraft-server:multiarch` since that includes support for ARM processors.

**Reply by wvendrig** [reply]
yes thanks this worked I'm impressed!
[08:41:59] [Worker-Main-7/INFO]: Preparing spawn area: 12%,
....
[08:42:46] [Worker-Main-7/INFO]: Preparing spawn area: 98%,
[08:42:47] [Server thread/INFO]: Time elapsed: 68384 ms,
[08:42:47] [Server thread/INFO]: Done (91.718s)! For help, type "help",
[08:42:47] [Server thread/INFO]: Starting remote control listener,
[08:42:47] [Server thread/INFO]: Thread RCON Listener started,
[08:42:47] [Server thread/INFO]: RCON running on 0.0.0.0:25575,
